aiida/restapi/run_api.py

```diff
@@ -143,11 +150,16 @@ def run_api(App, Api, *args, **kwargs):
         load_dbenv()
```

Comment:
I would probably remove this check. If load_dbenv() was already loaded, there is the probability that this happened with the wrong profile... So we should rather remove the check and make sure the function is never called when load_dbenv() is already called (e.g. we could be even more strict, and raise `if is_dbenv_loaded()`